### PR TITLE
asynclog

### PR DESCRIPTION
**Pull request by reshke**

### CODE REVIEW

File: sources/logger.c [hunk 1]
```diff
@@ -32,7 +32,52 @@ static int od_log_syslog_level[] = { LOG_INFO, LOG_ERR, LOG_DEBUG, LOG_CRIT };
...
+	(void)rc;
+}
+
+static inline void od_logger(void *arg)
+{
+	od_logger_t *logger = arg;
+
+	for (;;) {
+		machine_msg_t *msg;
+		msg = machine_channel_read(logger->task_channel, UINT32_MAX);
```
Comment (x4m) [inline sources/logger.c:57]: How are we going to shutdown? When Odyssey get SEGQUIT or SIGTERM?

File: sources/config_reader.c [hunk 2]
```diff
@@ -122,16 +126,22 @@ static od_keyword_t od_config_keywords[] = {
 	od_keyword("log_syslog_ident", OD_LLOG_SYSLOG_IDENT),
 	od_keyword("log_syslog_facility", OD_LLOG_SYSLOG_FACILITY),
 	od_keyword("stats_interval", OD_LSTATS_INTERVAL),
+	od_keyword("asynclog", OD_LLOG_ASYNC),
```
Comment (x4m) [inline sources/config_reader.c:129]: Is there any reason to disable this?

Comment (reshke) [reply sources/config_reader.c]: no reason for sure

File: sources/logger.c (hunk 1 above)
Comment (reshke) [reply sources/logger.c]: yes, just like other threads

**Pull request merged**